content/en/docs/instrumentation/java/automatic/withspan.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Use instrumentation annotations
+description: OpenTelemetry documentaion for using instrumentation annotations with Java agent.
+weight: 2
+---
+
+For most users, the out-of-the-box instrumentation is completely sufficient and nothing more has to
+be done.  Sometimes, however, users wish to create spans for their own custom code without doing too much code change.
+
+# Dependencies
+
+You'll need to add a dependency on the `opentelemetry-extension-annotations` library to 
+use the `@WithSpan` annotation.
+
+## Maven
+
+```xml
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-extension-annotations</artifactId>
+      <version>1.11.0</version>
+    </dependency>
+  </dependencies>
+```
+
+## Gradle
+
+```groovy
+dependencies {
+    implementation('io.opentelemetry:opentelemetry-extension-annotations:1.11.0')
+}
+```
+
+# Creating spans around methods with `@WithSpan`
+
+Another common situation is to capture a span corresponding to one of your methods. The
+`@WithSpan` annotation makes this straightforward:
+
+```java
+import io.opentelemetry.extension.annotations.WithSpan;
+
+public class MyClass {
+  @WithSpan
+  public void MyLogic() {
```

Comment:
`MyMethod` instead of `MyLogic`?

Reply:
Method names in Java should start with a lowercase letter, how about `myMethod()`?